### PR TITLE
Dev/smallchanges

### PR DESCRIPTION
**Pull request by 05saitejaswi**
Worked on the TotalCompletedTask ,when the status is completed then the task must come under TotalCompletedTask.

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
         var result;
         var totalUnCompletedTask = 0;
 
+
```
Comment (shivamgoyal7) [inline functions/index.js:35]: remove this extra space

File: public/index.html [hunk 2]
```diff
@@ -235,7 +234,13 @@ <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;"
 
     <div id="taskPage">
         <div class="container pt-2 pb-5 pl-2">
-            <h3 class="ml-1 fontHeading">Tasks</h3>
+            <div class="row pt-5">
+                <h3 class="col-md-8 col-8 fontHeading">Tasks</h3>
+
+                <span type="button" class="col-md-4 col-1  material-icons" id="backToMainFromCreateNewTask">
```
Comment (shivamgoyal7) [inline public/index.html:240]: change the id name.
And give some common back button id everywhere.

File: public/index.html [hunk 3]
```diff
@@ -235,7 +234,13 @@ <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;"
 
     <div id="taskPage">
         <div class="container pt-2 pb-5 pl-2">
-            <h3 class="ml-1 fontHeading">Tasks</h3>
+            <div class="row pt-5">
+                <h3 class="col-md-8 col-8 fontHeading">Tasks</h3>
+
```
Comment (shivamgoyal7) [inline public/index.html:239]: don't give extra space

File: public/scripts/config.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ var firebaseConfig = {
     measurementId: "G-FEQ8PFLLCV"
 };
 
-
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
```
Comment (shivamgoyal7) [inline public/scripts/config.js:12]: you can remove this commented line

File: public/scripts/frames.js [hunk 5]
```diff
@@ -9,6 +9,13 @@ function getTasks() {
 function getTasksData() {
     var result = "ok";
 
+    // if (selectedSprint = "-1") {
+    //     selectedDocument = "Backlog";
+    // } else {
+    //     selectedDocument = "S" + selectedSprint;
+    // }
+    selectedDocument = "S" + selectedSprint;
+
```
Comment (shivamgoyal7) [inline public/scripts/frames.js:18]: Update your branch and you get the correct changes here.

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
Update your branch.
After these changes start the remaining task

File: functions/index.js (hunk 1 above)
Comment (05saitejaswi) [reply functions/index.js:35]: k

File: public/scripts/config.js (hunk 4 above)
Comment (05saitejaswi) [reply public/scripts/config.js:12]: k

File: public/index.html (hunk 3 above)
Comment (05saitejaswi) [reply public/index.html:239]: k

File: public/scripts/worktrollyUI.js [hunk 6]
```diff
@@ -121,6 +121,8 @@ $("#submitCreateNewTask").click(function() {
     var category = $("#categoryCreateNewTask").val();
     var storyPointNumber = $("#storyPointNumber").val();
     var sprintNumber = $("#createNewTaskSprintNumber").val();
+    // var Clear = document.querySelector('submit');
+    // var inputs = document.querySelectorAll('input');
```
Comment (shivamgoyal7) [inline public/scripts/worktrollyUI.js:125]: remove commented line

File: public/scripts/frames.js [hunk 7]
```diff
@@ -8,7 +8,7 @@ function getTasks() {
 
 function getTasksData() {
     var result = "ok";
-
+    selectedDocument = "S" + selectedSprint;
```
Comment (shivamgoyal7) [inline public/scripts/frames.js:11]: remove this line no. 11

**Review by shivamgoyal7 — COMMENTED** [review]
check

File: public/scripts/worktrollyUI.js (hunk 6 above)
Comment (05saitejaswi) [reply public/scripts/worktrollyUI.js:125]: k

File: public/scripts/frames.js (hunk 7 above)
Comment (05saitejaswi) [reply public/scripts/frames.js:11]: k

**Review by shivamgoyal7 — APPROVED** [review]
able to merge

File: functions/index.js [hunk 8]
```diff
@@ -206,19 +205,53 @@ exports.logWork = functions.https.onRequest((request, response) => {
...
-                    Status: status
+            var updatePromise = db.collection(fullSprintId).doc(taskId).update({
+                LogWorkTotalTime: logWorkTotalTime,
+                WorkDone: workDone,
+                LogHours: logHours,
+                Status: status
+            });
+            return Promise.resolve(updatePromise);
+        })
+
```
Comment (viv9k) [inline functions/index.js:219]: Please remove empty line

**Review by viv9k — APPROVED** [review]
Ship it

**Review by shivamgoyal7 — COMMENTED** [review]
hey, saitejaswi one small error is there in your code, do it asap.

**Review by shivamgoyal7 — COMMENTED** [review]
do it asap

**Pull request merged**